### PR TITLE
Berna

### PR DESCRIPTION
**Pull request by bernasantiago**

### CODE REVIEW

**Review by cbanavi — APPROVED** [review]
lgtm!

File: src/berna_project/week1/finra.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                System.out.print(i + " ");
+            }
+
+        }
+
+    }
+
+}
+
+
```
Comment (jenneft) [inline src/berna_project/week1/finra.java:36]: great work

File: src/berna_project/week1/oddOrEven.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        if (n %2 == 0){
+            System.out.println("Even number.");
+        } else if (n %2 == 1) {
+            System.out.println("Odd number.");
+
+        }
+        return  n;
+
+    }
+}
```
Comment (jenneft) [inline src/berna_project/week1/oddOrEven.java:23]: all good

**Review by jenneft — APPROVED** [review]
Berna's job approved

**Pull request merged**